cmd/cloud/logger.go

```diff
@@ -1,13 +1,16 @@
 package main
 
 import (
+	"github.com/mattermost/mattermost-cloud/internal/model"
 	log "github.com/sirupsen/logrus"
 )
 
+var instanceID string
 var logger log.FieldLogger
 
 func init() {
-	logger = log.New()
+	instanceID = model.NewID()
```

Comment:
The thinking with the `instanceID` is to be able to uniquely identify a particular instance among others. We use this when locking clusters so that we can correlate "who locked" the cluster with logs from that instance. It should also help when we aggregate the logs from all the servers to pull out the instance-specific ones.

Reply:
Hmm, an excellent point -- we should indeed be locking per-worker instead of just per-instance.